### PR TITLE
feat: migrate to pnpm

### PR DESCRIPTION
**Pull request by GabsEdits**
Starting with the website

### CODE REVIEW

**Review by mirkobrombin — APPROVED** [review]

**Comment by mirkobrombin** [comment]
did you leave out recipe.yml on purpose?

**Review by mirkobrombin — CHANGES_REQUESTED** [review]
did you leave out recipe.yml on purpose?

**Comment by GabsEdits** [comment]
> did you leave out recipe.yml on purpose?

I somehow missed it. It's all good now, and I've added pnpm using corepack. It should work with no issues 😄

**Review by mirkobrombin — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "private": true,
   "scripts": {
     "dev": "node server",
-    "build": "npm run generate-articles && npm run build:client && npm run build:server",
+    "build": "pnpm run generate-articles && npm run build:client && npm run build:server",
```
Comment (muhdsalm) [inline package.json:8]: You left a few here.

**Review by muhdsalm — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (GabsEdits) [reply package.json:8]: Oh yeah, I missed them. I'll open a quick pr

Comment (GabsEdits) [reply package.json:8]: Opened: https://github.com/Vanilla-OS/website/pull/241

**Pull request merged**